clusterDeployer.py

```diff
@@ -200,6 +209,16 @@ def configure_bridge(h: host.Host, api_network: str) -> None:
   </ip>
 </network>
 """
+        else:
+            contents = """
+<network>
+  <name>default</name>
+  <forward mode='nat'/>
+  <bridge name='virbr0' stp='off' delay='0'/>
+  <ip address='192.168.123.250' netmask='255.255.0.0'/>
```

Comment:
You've copy pasted this from above and changed a small part of it. Can we have contents be equal to 
common_part
{specific_part}
remaining_common_part

and have the if else set the specific_part which is then formatted into the contents unconditionally?